core/services/ocr2/plugins/functions/integration_tests/v1/internal/testutils.go

```diff
@@ -578,6 +580,18 @@ func CreateFunctionsNodes(
 	return bootstrapNode, oracleNodes, oracleIdentites
 }
 
+// NOTE: This approach is technically incorrect because the returned port
+// can still be taken by the time the caller attempts to bind to it.
+// Unfortunately, we can't specify zero port in P2P.V2.ListenAddresses at the moment.
+func getFreePort(t *testing.T) uint16 {
```

Comment:
the function is a clear code duplicate, maybe worth moving it to global testutils?

Reply:
v0 directory will be removed soon. Also I indicated in the comment that this is not really a correct way to find an open port so I'd rather not put it in a shared lib.